native/screens/Settings.jsx

```diff
@@ -0,0 +1,12 @@
+import { Text, View } from 'react-native'
+import React, { Component } from 'react'
+
+export default class Settings extends Component {
```

Comment:
class-based components are slowly, but surely, becoming deprecated for react developers, please convert this to a functional arrow function